tests/registry.rs

```diff
@@ -252,26 +252,101 @@ fn package_with_path_deps() {
         .build();
 
     assert_that(p.cargo("package").arg("-v"),
-                execs().with_status(101).with_stderr_contains("\
```

Comment:
Shoud be the original error. This patch is not to change any behavior when `package.publish=true`.